README.md

```diff
@@ -15,91 +15,69 @@ and using:
 
-Due to the auto-generated nature of this collection of libraries, complete APIs or specific versions can appear or go away without notice.
-As a result, you should always locally vendor any API(s) that your code relies upon.
+[![Go Reference](https://pkg.go.dev/badge/google.golang.org/api.svg)](https://pkg.go.dev/google.golang.org/api)
 
-These client libraries are officially supported by Google.  However, the libraries are considered complete and are in maintenance mode. This means that we will address critical bugs and security issues but will not add any new features.
+These are auto-generated Go libraries from the Google Discovery Service's JSON description files.
 
-If you're working with Google Cloud Platform APIs such as Datastore or Pub/Sub,
-consider using the
-[Cloud Client Libraries for Go](https://github.com/googleapis/google-cloud-go)
-instead. These are the new and
-idiomatic Go libraries targeted specifically at Google Cloud Platform Services.
+Due to the auto-generated nature of this collection of libraries they may contain breaking changes from one release to
+the next.
 
-The generator itself and the code it produces are beta. Some APIs are
-alpha/beta, and indicated as such in the import path (e.g.,
-"google.golang.org/api/someapi/v1alpha").
+These client libraries are officially supported by Google.  However, the libraries are considered complete and are in
+maintenance mode. This means that we will address critical bugs and security issues but will not add any new features.
 
-## Application Default Credentials Example
+If you're working with Google Cloud Platform APIs such as Datastore or Pub/Sub, please use the
+[Cloud Client Libraries for Go](https://github.com/googleapis/google-cloud-go) instead. These are the new and idiomatic
+Go libraries targeted specifically at Google Cloud Platform Services.
 
-Application Default Credentials provide a simplified way to obtain credentials
-for authenticating with Google APIs.
+The generator itself and the code it produces are beta. Some clients are in alpha/beta and indicated as such in the
+import path (e.g., "google.golang.org/api/someapi/v1alpha").
 
-The Application Default Credentials authenticate as the application itself,
-which make them great for working with Google Cloud APIs like Storage or
-Datastore. They are the recommended form of authentication when building
-applications that run on Google Compute Engine or Google App Engine.
+## Authorization
 
-Default credentials are provided by the `golang.org/x/oauth2/google` package. To use them, add the following import:
+By default, each API will use [Google Application Default Credentials](https://developers.google.com/identity/protocols/application-default-credentials)
+for authorization credentials used in calling the API endpoints. This will allow your application to run in many
+environments without requiring explicit configuration.
 
 ```go
-import "golang.org/x/oauth2/google"
+client, err := storage.NewService(ctx)
```

Comment:
Probably don't use `storage` since that should be cloud.google.com/go/storage? Also, show the import?

Reply:
Good ideas. Switch to sheets as workspace apis are only found in this repo.